src/content/decentralized-identity.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Decentralized Identity 
+description: An overview of decentralized identity on Ethereum
+lang: en
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Centralized identity systems are fragmented, poorly secured, and offer minimal privacy protection
+summaryPoint2: Decentralized identity increases personal control of one's identity information.
+summaryPoint3: Decentralized ID projects built on Ethereum can revolutionize identity management. 
+---
+
+# What are identity, identifiers and attestations? {#what-are-identity-identifiers-attestations}
+
+## What is identity {#what-is-identity}
+
+Identity means an individual's sense of self, defined by a set of unique characteristics. Identity refers to the fact of being an _individual_, i.e., a distinct human entity. Identity could also refer to other non-human entities, such as an organization or authority.
+
+## What are identifiers? {#what-are-identifiers}
+
+An identifier is a piece of information that can be used to distinguish a person's identity. Common identifiers include the following: 
+- Name
+- Social security number/tax ID number 
+- Mobile number 
+- Date and place of birth
+- Digital identification credentials, e.g., passwords, email address, usernames
+
```

Comment:
Would introduce here the concept that these classic examples of identifiers are all issued, held and controlled by central entities:
ie
Name -- _Legal name_ issued by government, and cannot be issued/changed without permission
SSN/TIN -- Issued by government, cannot be issued/changed without permission
Mobile number -- Issued by phone company, a new number or changing your old number requires permission from phone company
Twitter/Facebook/Discord/Reddit/GitHub/etc handle -- Requires permission from platform to issue or change.

```suggestion

It is important to understand that in these traditional examples of identifiers that _issuance_ and _control_ over these identifiers are maintained by a central agency. For example, legal names require permission from a government for issuance or before being changed, and changes to your social media handle require permission from the platform. 

```

Reply:
This is a good point. Thanks for adding it.